src/lib/pickles_types/plonk_types.mli

```diff
@@ -247,6 +251,10 @@ module Evals : sig
     ; s : 'a Permuts_minus_1_vec.t
     ; generic_selector : 'a
     ; poseidon_selector : 'a
+    ; complete_add_selector : 'a
```

Comment:
Also picky, but accessors are added for the other selectors. Do we want to stay consistent?